dev/com.ibm.ws.security.oauth/resources/com/ibm/ws/security/oauth20/internal/resources/OAuthMessages.nlsprops

```diff
@@ -432,16 +436,16 @@ OAUTH_AUTH_HEADER_NOT_BASIC_AUTH.explanation=The specified request must contain
 OAUTH_AUTH_HEADER_NOT_BASIC_AUTH.useraction=Ensure that the request uses the Basic authentication scheme to include OAuth client credentials.
 
 # Do not translate internalClientId, internalClientSecret, isEnabled, appPasswordAllowed, or appTokenAllowed
-OAUTH_UI_ENDPOINT_NOT_ENABLED=CWWKS1494E: The request sent to the {0} URI is ignored because the URI is not enabled. One or more of the client or provider attributes are not set. Verify that the internalClientId and internalClientSecret provider attributes are configured. Verify that the isEnabled, appPasswordAllowed, or appTokenAllowed client attributes are configured.
+OAUTH_UI_ENDPOINT_NOT_ENABLED=CWWKS1494W: The request sent to the {0} URI cannot be processed because one or more of the client or provider attributes are not set. Verify that the internalClientId and internalClientSecret provider attributes are configured. Verify that the isEnabled, appPasswordAllowed, or appTokenAllowed client attributes are configured.
 OAUTH_UI_ENDPOINT_NOT_ENABLED.explanation=The client and provider attributes must be set for the URI to process requests.
 OAUTH_UI_ENDPOINT_NOT_ENABLED.useraction=Set the client and provider attributes correctly so the provider can process the request.
 
 # Java runtime level
-JAVA8_REQUIRED=CWWKS1495W: The functions available at the {0} URI require Java 8 or higher. The request is ignored.
+JAVA8_REQUIRED=CWWKS1495W: The OpenID Connect provider cannot process the [{0}] URI request because the Java version is lower than Java 8.  Java 8 or higher is required.
 JAVA8_REQUIRED.explanation=The Java version in  use is not 8 or higher.
 JAVA8_REQUIRED.useraction=Install Java version 8 or higher.
 
-JAVA8_REQUIRED_FOR_AT_HASHING=CWWKS1496W: The {0} OpenIdConnect provider has the {1} attribute set to use {2} algorithm, which requires Java 8 or higher. The specified provider ignores the {3} setting.
+JAVA8_REQUIRED_FOR_AT_HASHING=CWWKS1496W: The {0} OpenIdConnect provider has the {1} attribute set to use {2} algorithm and this requires Java 8 or higher. The specified provider will ignore the accessTokenEncoding setting.
```

Comment:
The appropriate styling is "OpenID Connect provider". And does the code that emits this message not have access to the name of the config attribute? Using an insert instead of hard coding the config attribute name in the message would be more flexible.

Reply:
fixed